src/main/java/com/flipkart/foxtrot/client/FoxtrotClientConfig.java

```diff
@@ -27,6 +27,21 @@
      */
     private int maxConnections = 10;
```

Comment:
We can increase maxConnections to 50. We will get blocked in case, any request receives events from more than 10 apps in a single kafka read batch

Reply:
this is default value, can be configured in foxtrot fabric processor topology properties